src/nvapi/nvapi_d3d_low_latency_device.h

```diff
@@ -47,6 +48,8 @@ namespace dxvk {
         inline static std::mutex m_lowLatencyDeviceMutex;
         inline static std::mutex m_lowLatencyFrameIdGeneratorMutex;
 
+        inline static bool m_lowLatencyMode;
```

Comment:
I don't think this should be `static`, or at least as long as it isn't some kind of map…

Consider this scenario, which should probably pass but it appears that with this code it likely won't:

```cpp
// assuming device1 != device2
NvAPI_D3D_SetSleepMode(device1, { .version = …, .bLowLatencyMode = false });
NvAPI_D3D_SetSleepMode(device2, { .version = …, .bLowLatencyMode = true });
NV_GET_SLEEP_STATUS_PARAMS status = { … };
NvAPI_D3D_GetSleepStatus(device1, &status);
REQUIRE_FALSE(status.bLowLatencyMode);
```

(Yet another argument towards making remaining device classes hold some instance data.)

Reply:
Done as a separate commit as we didn't tracked per device earlier. But agree, good idea to do so and indeed wise to rework this after this PR.